qemu/tests/numa_cpu.py

```diff
@@ -0,0 +1,149 @@
+import logging
+
+from virttest import error_context
+from virttest.utils_misc import NumaInfo
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Assign cpu to numa node with "-numa cpu", check the numa info in monitor
+    and guest os match with the qemu cli
+    """
+
+    def convert_cpu_topology_to_index(socket, die, core, thread):
```

Comment:
How about 'convert_numa_cpus_to_list'? It's only used to convert '-num cpus,xxx'.

Reply:
> How about 'convert_numa_cpus_to_list'? It's only used to convert '-num cpus,xxx'.

Eh~   I prefer `topology` a little more, since it is a common function to do topology and index(id) mapping.  The input can not only come from the `- numa cpu ***` cli.  `get_hotpluggable_cpus` can use it too, i just update to use it in `get_hotpluggable_cpus`.

I rename the function to  `convert_cpu_topology_to_id`, is it acceptable for you ?

Reply:
Updated with `ids` as we discussed offline, could you help review again, thanks!